ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/MH_SERVERS.yml

```diff
@@ -45,3 +45,26 @@ routers:
         switch_ports: [ Ethernet10 ]
         switches: [ MH-LEAF2A ]
         profile: Tenant_X_LT
+
+  ROUTER02_WITH_SUBIF:
+    adapters:
+      - endpoint_ports: [ Eth1, Eth2 ]
+        switch_ports: [ Ethernet11, Ethernet11 ]
+        switches: [ MH-LEAF1A, MH-LEAF1B ]
+        port_channel:
```

Comment:
I would add the description for this test, otherwise, the description is rendered ending with an underscore, which can lead to confusion.

Reply:
fixed